src/helpers/queryHelpers.ts

```diff
@@ -72,8 +72,8 @@ export const useServiceQuery = <TItem, TColumnKey extends string, TSortableColum
         setRefreshTime?.(new Date());
         return response.data;
       } catch (error) {
-        console.error(error);
-        throw error; // You can choose to throw the error or return a default value here
+        console.log('Error during data fetching:', error);
```

Comment:
Just an observation: `console.error()` is perfectly valid and supported by browsers.

[Camayoc has a special handler that will look for error messages](https://github.com/quipucords/camayoc/blob/main/camayoc/ui/client.py#L95) and consider a test failing if there are any. So moving that back to `console.error()` has a potential of failing the pipeline. But I think that's a good thing? As far as I can tell, we have not had any failures caused by `console.error()` here, so if it ever happens then probably this is something we should take a look at.

Reply:
i agree with the leveraging the `console.error` here. 

we do leverage `error` in a few places under the api calls but we also wrap some of them with a`!helpers.TEST_MODE` check. But this flag should only affect unit testing, not e2e since its running everything... maybe

Reply:
I will revert it back to console.error and let's see if it affects the pipeline at all.